utils/android/adb/commands.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import print_function
```

Comment:
Yes of course, I knew I had forgotten something. I'll add one to all the new Python files.